x-pack/plugin/monitoring/src/main/java/org/elasticsearch/xpack/monitoring/Monitoring.java

```diff
@@ -246,6 +252,51 @@ public UnaryOperator<Map<String, IndexTemplateMetadata>> getIndexTemplateMetadat
             map.remove(".monitoring-alerts");
             return map;
         };
+    }
+
+    @Override
+    public String getNodeStatsPluginName() {
+        return "monitoring";
+    }
+
+    @Override
+    public Stats getPluginNodeStats() {
+        return new MonitoringStats(Map.of("extra plugin", "example"));
```

Comment:
I assume this stuff is going to move to a test somewhere instead of the production version?